### PR TITLE
ENCD-5415 Fill in audits on Annotation metadata.tsv

### PR DESCRIPTION
**Pull request by forresttanaka**
Not sure if this is going into RC2 or v105. This is @zoldello’s code originally I think, so sending to him for code review.

### CODE REVIEW

**Comment by forresttanaka** [comment]
BTW the not-bdd-non-indexing test fails for a reason fixed in ENCD-5421.

File: src/encoded/batch_download.py [hunk 1]
```diff
@@ -282,11 +282,11 @@ def _get_annotation_metadata(request, search_path, param_list):
     writer = csv.writer(fout, delimiter='\t')
     writer.writerow(header)
     param_list['limit'] = ['all']
-    fields = [''.join(['&field=', str(value[0])]) for _, value in _tsv_mapping_annotation.items()]
-    path = '{}?{}{}'.format(
+    fields = ['field={}'.format(str(value[0])) for _, value in _tsv_mapping_annotation.items()] + ['field=audit']
```
Comment (zoldello) [inline src/encoded/batch_download.py:285]: f-string is easier to read. But this suffices.

**Review by zoldello — APPROVED** [review]
Code looks fine. My comment is more along the line of personal preference and has no significant effect.

**Comment by hitz** [comment]
circle-ci failure is fixed in another ticket, merging

**Pull request merged**